index.html

```diff
@@ -77,7 +77,24 @@ <h3>And now!..</h3>
 		
 	 
 	 
-	 <footer>Contact details:</footer>
-	  
+	 <footer class="wide-footer">
+	   <div class="contact-details">
+	     <h3>Contact details:</h3>
+	   </div>
+	   <div class="email">
+		 <h3>Email:</h3>
+		 <a href="mailto:timtapley@aol.com">timtapley@aol.com</a> 
+		 <br>
```

Comment:
Do you need this break here? We can usually achieve line breaks using css rather than the br tag.